Y2021_info.php

```diff
@@ -275,8 +275,9 @@
 		'Logic_Programming_with_Cut' => [
 			'spaceid' => 425941,
 			'type' => 'termination',
-			'id' => false,
```

Comment:
Thanks for registering! Is this change intended?

Reply:
I think this was a misunderstanding in our group. Besides these, all of the ids where `null`, so I said that if we add a new category we should write `null` instead of `false`. Thus, my colleague thought that he should also change these ids. I can change them back to `false` and then do a new pull-request if that helps.

Reply:
Ah I see, it should have been uniform from the beginning. I'll accept it and unify later.